api/management/commands/acquiredata.py

```diff
@@ -0,0 +1,29 @@
+import os
+from urllib.request import urlretrieve
+
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--path',
+            dest='path',
+            default='data',
+            help='Path to location of data files.',
+        )
+
+    def handle(self, *args, **options):
```

Comment:
@cgreene good to know. If the standard is to use `*args` here, keep it. Didn't realize this could get called in alternative ways.